packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -240,18 +244,20 @@ export default {
     computed: {
         ...mapState(VUEX_CHECKOUT_MODULE, [
             'address',
+            'basket',
             'customer',
+            'errors',
+            'geolocation',
             'id',
-            'isFulfillable',
+            'isFulfilable',
             'isLoggedIn',
+            'isable',
```

Comment:
This one looks like it's missing the start of the word possibly?

Reply:
🤦